src/ui/types/company-house-response.d.ts

```diff
@@ -15,8 +15,8 @@ interface CompanyHouseResponse {
   companyNumber: string;
   dateOfCreation: string;
   sicCodes: Array<string>;
-  success: boolean;
-  apiError: boolean;
+  success?: boolean;
+  apiError?: boolean;
```

Comment:
What's the reason for this change?

Reply:
the mapping from the database comes as  CompanyHouseResponse which does not have success or apiError and did not want to set a value from them.  Also used by the API which has those flags

Reply:
Could we use `ApplicationExporterCompany` type instead for these instances?

Reply:
I would need to remap the api response, do I do it in this ticket?

Reply:
I think so if it's not a big job, to avoid introducing tech debt